pytition/petition/management/commands/gen_sig.py

```diff
@@ -0,0 +1,53 @@
+import logging
+import random
+import string
+from django.core.management import BaseCommand
+
+from petition.models import Petition, Signature
+
+
+logger = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    """Generate signatures for petitions
+
+    ./manage.py gen_sig 1
+    > Generate a random signature for Petition:1
+    ./manage.py gen_sig "Petition Test" -n 10
+    > Generate 10 signatures for Petition:Petition Test if unique
+    """
+    def add_arguments(self, parser):
+        parser.add_argument('petition', type=str)
+        parser.add_argument('--number', '-n', type=int, default=1)
+
+    def handle(self, *args, **options):
+        try:
```

Comment:
I'm wondering if that would be better to just check with `if options['petition'].isnumeric():` instead of bouncing on the type of exception raised?